pom.xml

```diff
@@ -1532,6 +1541,23 @@
             <groupId>org.json</groupId>
             <artifactId>json</artifactId>
           </exclusion>
+          <!-- Hive 2.3.4 -->
+          <exclusion>
+            <groupId>${hive.group}</groupId>
+          <artifactId>hive-llap-tez</artifactId>
+          </exclusion>
+          <exclusion>
```

Comment:
Exclude `calcite-druid` and `avatica`. more details: https://issues.apache.org/jira/browse/SPARK-27054